lib/ie-contract.js

```diff
@@ -1,19 +1,16 @@
 import { ethers } from 'ethers'
-import { RPC_URL, rpcHeaders } from './config.js'
+import { rpcUrls, GLIF_TOKEN } from './config.js'
 import * as SparkImpactEvaluator from '@filecoin-station/spark-impact-evaluator'
-import pTimeout from 'p-timeout'
 
 export const createMeridianContract = async (contractAddress = SparkImpactEvaluator.ADDRESS) => {
-  const fetchRequest = new ethers.FetchRequest(RPC_URL)
-  fetchRequest.setHeader('Authorization', rpcHeaders.Authorization || '')
-  const provider = new ethers.JsonRpcProvider(
-    fetchRequest,
-    null,
-    {
+  const provider = new ethers.FallbackProvider(rpcUrls.map(url => {
+    const fetchRequest = new ethers.FetchRequest(url)
+    fetchRequest.setHeader('Authorization', `Bearer ${GLIF_TOKEN}`)
```

Comment:
Is it okay to send our GLIF_TOKEN to other RPC API providers?

Reply:
We also do this in spark-api: https://github.com/filecoin-station/spark-api/blob/main/api/lib/ie-contract.js#L7

We could make this conditional (`if (url.includes('glif'))`), although I also don't see an immediate risk with this.

Reply:
Fair enough 👍🏻 

:shipit: